### PR TITLE
skin/confluence: Add recording icons for currently recording channels to some skin locations/lists

### PR DESCRIPTION
**Pull request by nemphys**
skin/confluence: Add recording icons for currently recording channels to fullscreen osd, osdchannels and main channel list, add next programme on fullscreen osd.

### CODE REVIEW

File: addons/skin.confluence/720p/DialogPVRChannelsOSD.xml [hunk 1]
```diff
@@ -115,6 +115,14 @@
 						<aligny>center</aligny>
 						<label>$INFO[ListItem.Label]</label>
 					</control>
+					<control type="image">
+                                                <posx>5</posx>
```
Comment (opdenkamp) [inline addons/skin.confluence/720p/DialogPVRChannelsOSD.xml]: please correct the identing throughout this patch

File: addons/skin.confluence/720p/DialogPVRChannelsOSD.xml [hunk 2]
```diff
@@ -260,4 +276,4 @@
 			</control>
 		</control>
 	</controls>
-</window>
```
Comment (opdenkamp) [inline addons/skin.confluence/720p/DialogPVRChannelsOSD.xml]: don't don't cosmetic edits like this please when you're sending a patch to add something. cosmetic edits are fine, but stick them in a separate commit so I can review them and ignore them if I don't like the change.

File: addons/skin.confluence/720p/VideoFullScreen.xml [hunk 3]
```diff
@@ -159,6 +159,13 @@
 					<height>35</height>
 					<itemgap>0</itemgap>
 					<orientation>horizontal</orientation>
+					<control type="image" id="1">
```
Comment (opdenkamp) [inline addons/skin.confluence/720p/VideoFullScreen.xml]: why the id=1?

**Comment by opdenkamp** [comment]
to summarise: please remove the cosmetic edits or stick them in separate commits and correct the identing.

thanks!

File: addons/skin.confluence/720p/DialogPVRChannelsOSD.xml (hunk 1 above)
Comment (nemphys) [inline addons/skin.confluence/720p/DialogPVRChannelsOSD.xml]: Sorry, don't have a clue how this happened and I didn't notice

File: addons/skin.confluence/720p/VideoFullScreen.xml (hunk 3 above)
Comment (nemphys) [inline addons/skin.confluence/720p/VideoFullScreen.xml]: A leftover from testing, will correct

File: addons/skin.confluence/720p/DialogPVRChannelsOSD.xml (hunk 2 above)
Comment (nemphys) [inline addons/skin.confluence/720p/DialogPVRChannelsOSD.xml]: Right

**Comment by nemphys** [comment]
OK, looks fine to me now, please take a look when yo can

**Comment by opdenkamp** [comment]
there's still a cosmetic edit in a non-related file present, but I'll stop bitching cause it's minor :)

**Comment by nemphys** [comment]
Right, I was wondering all this time how you missed that ;-)

**Comment by nemphys** [comment]
OK, you're gonna love this, just realized that 'id="1"' was needed after all, without it it doesn't display the image....
I am making a new commit to fix this

**Pull request merged**